yt_dlp/extractor/tiktok.py

```diff
@@ -307,7 +310,8 @@ def extract_addr(addr, add_meta={}):
         contained_music_author = traverse_obj(
             music_info, ('matched_song', 'author'), ('matched_pgc_sound', 'author'), 'author', expected_type=str)
 
-        is_generic_og_trackname = music_info.get('is_original_sound') and music_info.get('title') == 'original sound - %s' % music_info.get('owner_handle')
+        is_generic_og_trackname = music_info.get('is_original_sound') and music_info.get(
+            'title') == 'original sound - %s' % music_info.get('owner_handle')
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
        is_generic_og_trackname = music_info.get('is_original_sound') and music_info.get('title') == 'original sound - %s' % music_info.get('owner_handle')
```